sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -38,6 +39,7 @@ internal class ViewAnnotationManagerImpl(
   internal val idLookupMap = ConcurrentHashMap<View, String>()
   private val currentlyDrawnViewIdSet = mutableSetOf<String>()
   private val unpositionedViews = mutableSetOf<View>()
+  private val annotationsMap = ConcurrentHashMap<View, ViewAnnotationOptions>()
```

Comment:
There's already `private val annotationMap = ConcurrentHashMap<String, ViewAnnotation>()` at line 37, I'd suggest to use different name here)

Reply:
Btw - do we need this separate field at all? Can we just get the views from `idLookupMap` and map them with `ViewAnnotationOptions` using `getViewAnnotationOptionsByView` ?

Reply:
updated.